### PR TITLE
fixing the fluent-bit-ib

### PR DESCRIPTION
**Pull request by Rajat0002**

### CODE REVIEW

File: community_images/fluent-bit/ironbank/k8s_coverage.sh [hunk 1]
```diff
@@ -12,17 +12,18 @@ SCRIPTPATH="$( cd -- "$(dirname "$0")" >/dev/null 2>&1 ; pwd -P )"
...
 echo "NAMESPACE: $NAMESPACE"
 echo "RELEASE_NAME: $RELEASE_NAME"
 
 sleep 10
-CONTAINER_NAME=$(kubectl get pods -n "$NAMESPACE" -l "app.kubernetes.io/instance=$RELEASE_NAME" -o jsonpath='{.items[0].metadata.name}')
-# copy over the script to the pod
-kubectl cp "${SCRIPTPATH}"/config/fluent-bit.config "${CONTAINER_NAME}":/tmp/fluent-bit.config -n "${NAMESPACE}"
-kubectl exec "${CONTAINER_NAME}" -n "${NAMESPACE}" -- /bin/bash -c "nohup /fluent-bit/bin/fluent-bit -c /tmp/fluent-bit.config" &
+
+CONTAINER_NAME="$RELEASE_NAME"
```
Comment (AnmolVirdi) [inline community_images/fluent-bit/ironbank/k8s_coverage.sh:24]: Please verify if this name is correct. Refer other images.

File: community_images/fluent-bit/ironbank/overrides.yml [hunk 2]
```diff
@@ -7,8 +7,6 @@ containerSecurityContext:
   allowPrivilegeEscalation: true
   capabilities:
     add: ["SYS_PTRACE"]
-resourceType: deployment
```
Comment (AnmolVirdi) [inline community_images/fluent-bit/ironbank/overrides.yml:10]: Any specific reason for removing this?

Comment (Rajat0002) [reply community_images/fluent-bit/ironbank/overrides.yml:10]: The official helm_chart required replica_count 1 for deployment resource type but not generally required

Comment (AnmolVirdi) [reply community_images/fluent-bit/ironbank/overrides.yml:10]: Can we just keep it then?

File: community_images/fluent-bit/ironbank/k8s_coverage.sh (hunk 1 above)
Comment (Rajat0002) [reply community_images/fluent-bit/ironbank/k8s_coverage.sh:24]: It is used both as "$RELEASE_NAME" and "$RELEASE_NAME"-0

File: community_images/fluent-bit/ironbank/overrides.yml (hunk 2 above)
Comment (Rajat0002) [reply community_images/fluent-bit/ironbank/overrides.yml:10]: No need as I tried using other helm chart where replica count was already there as 1 for deployment type in values.yml

**Pull request merged**